OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -66,6 +67,20 @@
   } while (0 && (assertion))
 #endif
 
+// Internal.
+#if _WIN32
```

Comment:
ifdef?

Reply:
oh, yes, I changed it to #if defined(_WIN32) for consistency with code above, thanks.